autoPyTorch/evaluation/abstract_evaluator.py

```diff
@@ -249,28 +250,29 @@ def __init__(self, backend: Backend,
             'X_test': self.X_test,
             'y_test': self.y_test,
             'backend': self.backend,
-            'logger_port': logger_port
+            'logger_port': logger_port,
         })
-
-        default_pipeline_options = self.pipeline_class.get_default_pipeline_options()
-
-        self.budget_type = default_pipeline_options['budget_type'] if budget_type is None else budget_type
-        self.budget = default_pipeline_options[self.budget_type] if budget == 0 else budget
-        self.fit_dictionary = {**default_pipeline_options, **self.fit_dictionary}
+        pipeline_config = pipeline_config if pipeline_config is not None\
+            else self.pipeline_class.get_default_pipeline_options()
+        self.budget_type = pipeline_config['budget_type'] if budget_type is None else budget_type
+        self.budget = pipeline_config[self.budget_type] if budget == 0 else budget
+        self.fit_dictionary = {**pipeline_config, **self.fit_dictionary}
 
         self.num_run = 0 if num_run is None else num_run
 
         logger_name = '%s(%d)' % (self.__class__.__name__.split('.')[-1],
                                   self.seed)  # TODO: Add name to dataset class
         if logger_port is None:
             logger_port = logging.handlers.DEFAULT_TCP_LOGGING_PORT
-        self.logger = get_named_client_logger(output_dir=self.backend.temporary_directory, name=logger_name,
-                                              port=logger_port)
+        self.logger = get_named_client_logger(
+            name=logger_name,
+            port=logger_port,
+        )
         self.Y_optimization: Optional[np.ndarray] = None
         self.Y_actual_train: Optional[np.ndarray] = None
         self.pipelines: Optional[List[BaseEstimator]] = None
         self.pipeline: Optional[BaseEstimator] = None
-        self.logger.debug("Fit dictionary in Abtract evaluator: {}".format(self.fit_dictionary))
+        # self.logger.debug("Fit dictionary in Abtract evaluator: {}".format(self.fit_dictionary))
```

Comment:
Is this too much output? I think for debug we want as much as possible

Reply:
yeah It was printing the whole X_train and y_train multiple times over, but I guess fit dictionary is important for debug, so, I'll put everything except the dataset keys to be printed